### DISCUSSION TITLE
Wildlife info from reserve map screen

### DISCUSSION THREAD

**Discussion by ResidentKrockhead** [janstehno/cotw-companion#62] [Ideas]
It would save a lot of time if I could open an animal’s information from a reserve’s map screen. (The menu you use to highlight drink/feed/rest areas would make sense. Perhaps a little expand option on the right side of each animals column linking to their information page so you don’t have to go back and forth so much.

**Comment by janstehno** [comment]
I was wondering who will come up with this as I always thought that this is the connection that is missing in the map feature. A possible solution (like you mentioned) would be to redirect the user from the list where animal zones can be activated to the animal info page. Other ideas of how I could do this differently are always welcome.

**Comment by janstehno** [comment]
Added in 1.10.6
